src/main/java/apoc/refactor/GraphRefactoring.java

```diff
@@ -114,6 +113,91 @@
         return doCloneNodes(nodes, true, Collections.emptyList());
     }
 
+    /**
+     * this procedure clones a subgraph defined by a list of nodes and relationships. The resulting clone is a disconnected subgraph,
+     * with no relationships connecting with the original nodes, nor with any other node outside the subgraph clone.
+     * This can be overridden by supplying a list of node pairings to reanchor relationships in the subgraph clone to other nodes in the graph.
+     * This is useful when instead of cloning a certain node or set of nodes, you want to instead reanchor relationships in the resulting clone
+     * such that they point to some different existing node in the graph.
+     *
+     * For example, this could be used to clone a branch from a tree structure (with none of the new relationships going
+     * to the original nodes) and to reanchor any relationships from an old root node (which will not be cloned) to a different existing root node.
+     *
+     */
+    @Procedure(mode = Mode.WRITE)
+    @Deprecated
+    @Description("apoc.refactor.cloneSubgraph([node1,node2,...], [rel1,rel2,...], [reanchorNodePairs = [] ], [skipProperties = [] ]) YIELD input, output, error | " +
+            "clone nodes with their labels and properties (optionally skipping specific properties), and clone the given relationships (will exist between cloned nodes only or be reanchored according to reanchor node pairings)")
+    public Stream<NodeRefactorResult> cloneSubgraph(@Name("nodes") List<Node> nodes,
+                                                    @Name("rels") List<Relationship> rels,
+                                                    @Name(value = "reanchors", defaultValue = "[]") List<List<Node>> reanchors,
+                                                    @Name(value = "skipProperties", defaultValue = "[]") List<String> skipProperties) {
+        Map<Node, Node> copyMap = new LinkedHashMap<>(nodes.size());
```

Comment:
is the order important? otherwise HashMap is good enough.

Reply:
Right, when I started implementation I thought I would need to iterate over the map's contents, but that's not happening, so HashMap it is.